DotNet/DataAcquisition/Dockerfile

```
@@ -1,14 +1,6 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/aspnet:8.0-jammy-amd64 AS base
-RUN apt-get update && apt-get install -y curl
-USER 1001
-WORKDIR /app
-ENV \
-    DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false \
-    LC_ALL=en_US.UTF-8 \
-    LANG=en_US.UTF-8
-
+FROM lantanagroup/aspnet8-run AS base
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)